src/components/panel/split_panel/split_panel.tsx

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { FunctionComponent, ReactNode, HTMLAttributes } from 'react';
+import classNames from 'classnames';
+import { EuiPanel, _EuiPanelProps, EuiPanelProps } from '../panel';
+import { EuiBreakpointSize } from '../../../services/breakpoint';
+import { useIsWithinBreakpoints } from '../../../services/hooks';
+
+export type _EuiSplitPanelInnerProps = HTMLAttributes<HTMLDivElement> &
+  Omit<
+    _EuiPanelProps,
+    | 'hasShadow'
+    | 'hasBorder'
+    | 'borderRadius'
+    | 'betaBadgeLabel'
+    | 'betaBadgeTooltipContent'
+    | 'betaBadgeTitle'
+  >;
+
+/**
+ * Consumed via `EuiSplitPanel.Inner`.
+ * Extends most `EuiPanelProps`.
+ */
+export const _EuiSplitPanelInner: FunctionComponent<_EuiSplitPanelInnerProps> = ({
+  children,
+  className,
+  ...rest
+}) => {
+  const classes = classNames('euiSplitPanel__inner', className);
+
+  const panelProps: _EuiPanelProps = {
+    hasShadow: false,
+    color: 'transparent',
+    borderRadius: 'none',
+    hasBorder: false,
+  };
+
+  return (
+    <EuiPanel className={classes} {...panelProps} {...(rest as EuiPanelProps)}>
+      {children}
+    </EuiPanel>
+  );
+};
+
+export type _EuiSplitPanelOuterProps = HTMLAttributes<HTMLDivElement> & {
+  /**
+   * Any number of #EuiSplitPanelInner components
```

Comment:
```suggestion
   * Any number of #_EuiSplitPanelInner components
```

Link doesn't work currently. This might fix it?

Reply:
It looks like when I add the `_` it also breaks the rendering as a link completely. But probably a more appropriate comment anyway